### PR TITLE
Add alert to Push Settings button on Search Page

### LINKED ISSUE #89: Push Settings button is dangerous!
The Push Setting button at the plugin Search Page can erase all settings made at Algolia control panel if pressed. So it needs to alert about this before someone pushes that button.

I had made some configurations at Algolia and then clicked on that "Push Settings" button to see what it does. Nothing happend! But then I look at Algolia on the configuration tab and saw that all my configurations is lost. 😅

### PR DESCRIPTION
**Pull request by asharirfan**
### Description

This PR fixes #89 

### Screenshot

image

### CODE REVIEW

**Comment by tw2113** [comment]
I agree overall, and like what I see for implementation. I'm wondering about messaging though. I say that in the way that the plugin is supposed to be able to push configurations set via code customizations and hooks and the like. So it's not guaranteed to be erasing settings from the dashboard, but just pushing a set version of the settings. Any parts that aren't set, I believe end up falling to defaults. If I use the dashboard to set categories as attributes for faceting, and then in code set that to categories and tags and click the button, the dashboard is going to update to the latter.

However, we can't get too wordy in a modal popup either. Closest/best wording replacement I can think of is "override".

Thoughts?

**Comment by asharirfan** [comment]
@tw2113 Agreed. I have made changes to the text.

image

File: includes/admin/class-algolia-admin.php [hunk 1]
```diff
@@ -99,6 +100,23 @@ public function enqueue_scripts() {
...
+	 * @author WebDevStudios <contact@webdevstudios.com>
+	 * @since  2.2.0-dev
+	 */
+	public function localize_scripts() {
+
+		wp_localize_script(
+			'algolia-admin-push-settings-button',
+			'algoliaPushSettingsButton',
+			array(
+				'pushBtnAlert' => __( 'Warning: Pushing settings will override the settings in the Algolia dashboard. Do you want to continue?', 'wp-search-with-algolia' ),
```
Comment (tw2113) [inline includes/admin/class-algolia-admin.php:115]: one last small detail, can we get this updated to use one of the escaping internationalization functions? I think `esc_html__` would be fine.

Comment (asharirfan) [reply includes/admin/class-algolia-admin.php:115]: Oh geez! I should have added that. Thanks for pointing this out!

**Review by tw2113 — APPROVED** [review]

**Comment by tw2113** [comment]
Good to go @asharirfan
